src/components/IconButton/index.jsx

```diff
@@ -33,10 +50,12 @@ const IconButton = ({
         .toLowerCase()}-button`}
       onClick={handleClick}
     >
-      <img
-        src={confirmationRequested ? questionMarkIcon : imgSrc}
-        alt={name}
-      />
+      <img src={getIcon()} alt={name} />
+      <span>
+        {confirmationRequested
+          ? "Confirm"
+          : name.replace(" ", "\u00A0")}
```

Comment:
why do we need to replace spaces in the names? This is the kind of line that's probably worth either refactoring out of the app or leaving a comment to explain.

Reply:
Good call, I'll pull this out into its own function. It's replacing spaces with the non-breaking space character.